doc/cephadm/services/mgmt-gateway.rst

```diff
@@ -80,7 +85,8 @@ To deploy the mgmt-gateway in a high availability setup, here is an example of t
       enable_auth: true
       virtual_ip: 192.168.100.220
 
-`Ingress` Configuration for Keepalived:
+In addition, the user must configure an `Ingress` service to provide virtual IP functionality for
+the `mgmt-gateway`. Below is a configuration example for the ingress service:
```

Comment:
NIT, I think what this is an example of is self-explanatory given the previous sentence and what's actually in the example
```suggestion
the `mgmt-gateway`. For example:
```

Reply:
fixed